packages/styles/gulpfile.js

```diff
@@ -136,7 +136,7 @@ gulp.task('sass', () => {
  */
 gulp.task('build-components', () => {
   return gulp
-    .src('./src/components/*.scss')
+    .src('./src/**/*.scss')
```

Comment:
@gfellerph It looks like the `sass` and `build-components` tasks are now the same, no?

Reply:
True, it can be removed.